doc/start/quick-cephfs.rst

```diff
@@ -3,117 +3,116 @@
 ===================
 
 To use the :term:`CephFS` Quick Start guide, you must have executed the
-procedures in the `Storage Cluster Quick Start`_ guide first. Execute this quick
-start on the Admin Host.
+procedures in the `Storage Cluster Quick Start`_ guide first. Execute this
+quick start on the admin host.
 
 Prerequisites
 =============
 
-#. Verify that you have an appropriate version of the Linux kernel. 
+#. Verify that you have an appropriate version of the Linux kernel.
    See `OS Recommendations`_ for details. ::
-   
+
 	lsb_release -a
 	uname -r
 
-#. On the admin node, use ``ceph-deploy`` to install Ceph on your 
+#. On the admin node, use ``ceph-deploy`` to install Ceph on your
    ``ceph-client`` node. ::
 
 	ceph-deploy install ceph-client
 
+#. Optionally, if you want a FUSE-mounted file system, you would need to
+   install ``ceph-fuse`` package as well.
 
 #. Ensure that the :term:`Ceph Storage Cluster` is running and in an ``active +
    clean``  state. Also, ensure that you have at least one :term:`Ceph Metadata
-   Server` running. :: 
+   Server` running. ::
 
 	ceph -s [-m {monitor-ip-address}] [-k {path/to/ceph.client.admin.keyring}]
 
 
-Create a Filesystem
-===================
+Create a File System
+====================
 
 You have already created an MDS (`Storage Cluster Quick Start`_) but it will not
-become active until you create some pools and a file system.  See :doc:`/cephfs/createfs`.
+become active until you create some pools and a file system.  See
+:doc:`/cephfs/createfs`.::
 
-::
-
-    ceph osd pool create cephfs_data <pg_num>
-    ceph osd pool create cephfs_metadata <pg_num>
+    ceph osd pool create cephfs_data 32
+    ceph osd pool create cephfs_metadata 32
     ceph fs new <fs_name> cephfs_metadata cephfs_data
 
+Quick word about Pools and PGs
+------------------------------
 
-Create a Secret File
-====================
-
-The Ceph Storage Cluster runs with authentication turned on by default. 
-You should have a file containing the secret key (i.e., not the keyring 
-itself). To obtain the secret key for a particular user, perform the 
-following procedure: 
+Since the default replication number/size is 3, you'd need 3 OSDs to get
+``active+clean`` for all PGs. Alternatively, you may change the replication
+number for the pool to match the number of OSDs::
```

Comment:
Consider mentioning erasure coding as an option here, too.

Reply:
Added an subsection that talks about erasure coded pools.  Mostly the information is from here - https://ceph.io/community/new-luminous-erasure-coding-rbd-cephfs/.

@ceph/cephfs Please verify that the wording and details are correct since I wasn't this much aware of EC coded pools until now.